docs/tutorial/getting-started.md

```diff
@@ -27,10 +27,10 @@ juju add-model wordpress-tutorial
 
 ### Deploy wordpress-k8s charm
 
-Deployment of WordPress requires a database integration. The preferred method in juju is through an
-integration. In this case, a `mysql` [interface](https://juju.is/docs/sdk/integration) is required
-by the wordpress-k8s charm and hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be
-used.
+Deployment of WordPress requires a database integration. The preferred method to provide it 
+in juju is through a elation. In this case, a `mysql` [interface](https://juju.is/docs/sdk/integration)
```

Comment:
We've switched from saying an integration here to a relation (or elation)

Reply:
Ok, so which one of the two "integraiton" appearances should we reword, and how?